lib/iris/coord_systems.py

```diff
@@ -840,17 +842,33 @@ def __init__(self, longitude_of_projection_origin=0, ellipsoid=None):
                     True longitude of planar origin in degrees.
             * ellipsoid
                     :class:`GeogCS` defining the ellipsoid.
+            * min_latitude
+                    the maximum southerly extent of the projection.
+                    Defaults to -80 degrees.
+            * max_latitude
+                    the maximum northerly extent of the projection.
+                    Defaults to 84 degrees.
+            * true_scale_lat
```

Comment:
Is this the name used by CF? Could you reference these in your description to speed up xreferencing please?

Reply:
Looks like CF accepts one of two names: http://cfconventions.org/cf-conventions/cf-conventions.html#_mercator. I'm going to go with `scale_factor_at_projection_origin` as it's the more similar to the use-case we're targetting.